### PR TITLE
Feature/develop design progress

### PR DESCRIPTION
**Pull request by biocom**

### CODE REVIEW

**Comment by biocom** [comment]
Screenshot 2020-02-18 at 14 58 54
Screenshot 2020-02-18 at 14 58 47
Screenshot 2020-02-18 at 14 58 35
Screenshot 2020-02-18 at 14 58 24

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr565--dexreact.review.gnosisdev.com

File: src/components/Layout/Header/Navigation.styled.ts [hunk 1]
```diff
@@ -51,9 +56,9 @@ export const NavLinksWrapper = styled.div<{ $open?: boolean; $responsive: boolea
       color: var(--color-text-active);
       background: var(--color-background-nav-active);
       @media ${MEDIA.mobile} {
-        padding: 0 1.2rem;
+        /* padding: 0 1.2rem;
         font-size: 1.6rem;
-        height: 4.6rem;
+        height: 4.6rem; */
```
Comment (alfetopito) [inline src/components/Layout/Header/Navigation.styled.ts]: Can you remove dead code?

File: src/components/UserWallet/UserWallet.styled.ts [hunk 2]
```diff
@@ -57,35 +141,114 @@ export const EtherImage = styled.img`
...
+      color: #526877;
+      font-weight: lighter;
+      opacity: 0.5;
+    }
+}
 `
 
 export const NetworkTitle = styled.div`
+  /* color: ${({ color = 'var(--color-text-primary)' }): string => color};
+  font-weight: bolder; */
```
Comment (alfetopito) [inline src/components/UserWallet/UserWallet.styled.ts]: Remove?

Comment (biocom) [reply src/components/UserWallet/UserWallet.styled.ts]: Removed.

File: src/components/Layout/Header/Navigation.styled.ts (hunk 1 above)
Comment (biocom) [reply src/components/Layout/Header/Navigation.styled.ts]: Done.

**Comment by biocom** [comment]
> Looks good. Just suggest to remove some dead code.

Updated now.

**Comment by Velenir** [comment]
Looks great!!

Would be cool if UserWallet didn't change height between `Connect -> Connecting -> Address` states, so that the page didn't jump on resize

**Comment by Velenir** [comment]
Btw, don't have to care about minifying svg, it's part of the pipeline

**Comment by biocom** [comment]
> Looks great!!
> 
> Would be cool if UserWallet didn't change height between `Connect -> Connecting -> Address` states, so that the page didn't jump on resize

Will optimise for that.

> Btw, don't have to care about minifying svg, it's part of the pipeline
Yes, though I didn't like the bulky src files :)

**Comment by biocom** [comment]
@alfetopito I ran `npm run lint:fix` and fixed some unused vars. The other issues either were present already or I can't fix them.

**Comment by W3stside** [comment]
getting some overflow hidden in the `orders` also would be nice to have the orders `overflow-y: auto`

image

**Comment by biocom** [comment]
@W3stside Today I'm going to touch the orders/balances again.

**Comment by W3stside** [comment]
also on the nav buttons - adding `outline: none` so it doesn't make those squares on click would be sweet

**Comment by biocom** [comment]
> also on the nav buttons - adding `outline: none` so it doesn't make those squares on click would be sweet

How can I replicate? I tested both desktop/mobile responsive Brave/Chrome and iOS simulator (Safari).

**Comment by W3stside** [comment]
> > also on the nav buttons - adding `outline: none` so it doesn't make those squares on click would be sweet
> 
> How can I replicate? I tested both desktop/mobile responsive Brave/Chrome and iOS simulator (Safari).
Screenshot from 2020-02-19 16-41-35

**Comment by W3stside** [comment]
what is the end goal of this PR? is this all the necessary style changes or just some? Should we not approve until the end?

**Comment by biocom** [comment]
It was meant as some intermediary update. But I'm fine to keep it open and update things on my end.

**Comment by alfetopito** [comment]
> what is the end goal of this PR? is this all the necessary style changes or just some? Should we not approve until the end?

> It was meant as some intermediary update. But I'm fine to keep it open and update things on my end.

I'd be happy to merge partial updates. A single huge PR would make it much harder to review.

**Comment by anxolin** [comment]
> I'd be happy to merge partial updates. A single huge PR would make it much harder to review.
+1 

Additionally. For the partials, it's also good to scope, so reviewers know what they should be reviewed. Normally we add a short description/picture/list in the description

looks amazing the progress by the way

**Pull request merged**